proteus

```diff
@@ -148,7 +148,13 @@ class WriteComposeFile:
         files = glob.glob("/dev/xclmgmt*")
         for file in files:
             devices.append(f"{file}:{file}")
-
+        # migraphx devices, if found
+        kfd = glob.glob('/dev/kfd')
```

Comment:
Do you know if you need all the files in these directories or some known subset? Just wondering if it's safe to add everything in these directories. I added the other device files only if their name matched some format e.g. render*

Reply:
globbing a single file gives a list with 1 member if it exists.  But this code creates a syntax bug and I'm pushing a fix